examples/nlp/text_extraction_with_bert.py

```diff
@@ -0,0 +1,318 @@
+"""
+Title: BERT for Text Extraction
```

Comment:
Oh okay. By the way I am very interested in writing examples of BERT and GPT with just `tf.keras`. Maybe just the pertaining part for example. We can discuss over over a separate issue if you think it would be a good addition.